test/core.controller.tests.js

```diff
@@ -879,4 +879,59 @@ describe('Chart.Controller', function() {
 			expect(chart.tooltip._options).toEqual(jasmine.objectContaining(newTooltipConfig));
 		});
 	});
+
+	describe('event handling', function() {
```

Comment:
do you think the responsive tests should move here as well? or just the `acquireContext`, `releaseContext` and event tests?